AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponentsLevel_DisplayMapperAdded.py

```diff
@@ -89,7 +148,7 @@ def AtomEditorComponentsLevel_DisplayMapper_AddedToEntity():
         Report.result(Tests.creation_redo, EditorLevelEntity.has_component(AtomComponentProperties.display_mapper()))
 
         # 4. Set LDR color Grading LUT asset.
-        display_mapper_asset_path = os.path.join("TestData", "test.lightingpreset.azasset")
+        display_mapper_asset_path = os.path.join("LookupTables", "LUT_Sepia.azasset")
```

Comment:
Is this specific to this component or will it affect other components? I see other paths with `"TestData"` used.

Reply:
TestData is used by some Atom tests, it is specified as an AssetProcessor target path for AutomatedTesting project so TestData is available. The issue here was that lightingpreset is not a valid asset to use in a display mapper which requires a LUT. The lightingpreset would load, but I believe a non-fatal assert would indicate issues existed.
This change assigns a valid LUT so there should be no question if the test is failing as to the asset type used.